bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -930,13 +928,14 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 
 <!-- 1850 for CLM5.1/CLM6.0 -->
 <init_interp_attributes sim_year="1850" use_cndv=".false." use_fates=".false." phys="clm5_1"
```

Comment:
@slevis-lmwg and I went over this a bit. This is in place when the namelist matching can't find an exact matching finidat based on all the details (resolution, etc.). It then wants to know the specifics of which finidat file it should match to.  So this often needs to be coordinated when a new finidat file is added. In this case we previously wanted to look for a ne30pg3 resolution finidat file, but now we want f09 since that's what we spun up in this case.